src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -399,6 +422,8 @@ def get_metadata_config(
         Args:
             expand_certs: Whether to expand the certificate paths to their
                 contents.
+            external: Whether to return the external metadata configuration
+                that is advertised to clients.
```

Comment:
I don't see any arg like this

Reply:
Thanks for noticing, docstring linter was complaining too. Removed it.